### DISCUSSION TITLE
Libre Le Potato (for Nixie)

### DISCUSSION THREAD

**Discussion by lrbrise** [tagyoureit/nodejs-poolController#585] [Nixie]
Does anyone know if the Libre Le Potato is compatible with the Raspberry Pi?
I'd like to use the hats supplied for Raspberry Pi.  The boards themselves look identical

**Comment by tagyoureit** [comment]
The [Kickstarter page](https://www.kickstarter.com/projects/librecomputer/libre-computer-board-next-gen-4k-sbc-dev-board-for#) says "drop in replacement" but then further down it says "Software and images made for Raspberry Pi must be modified to work". And on the same path, "Most Raspberry Pi HATs can be supported by the hardware but the software bits to get them working on this board is simply not there yet." 

I would guess eventually there will be ootb support, but not likely from Day 1.

**Reply by arrmo** [reply]
Agreed. The latest Ubuntu (v22.04 LTS) installs and runs, and it has the 5.19 kernel. So all good there. Seems a pinout check may be needed.

Do you happen to know, what interfaces are used on the hats (i.e. only I2C, or also SPI and/or GPIO)? Some digging yet 😆.

Thanks!

**Reply by tagyoureit** [reply]
Sequent hats and Atlas boards are i2c.

**Comment by arrmo** [comment]
Yep, looks just like an I2C bus, set the address per hat. Perfect!

One INT needed, but that pin is the same on RPi or Libre. Let's see, HW being ordered 😆.

Thanks!

**Reply by tagyoureit** [reply]
Oh geeze, totally missed that Kickstarter page was from 2017. Lol.
